src/wrapper.ts

```diff
@@ -16,6 +16,7 @@ import LumigoMongoDBInstrumentation from './instrumentations/mongodb/MongoDBInst
 import { extractEnvVars, getMaxSize, isEnvVarTrue, logger } from './utils';
 import * as awsResourceDetectors from '@opentelemetry/resource-detector-aws';
 import { AwsEcsDetector, LumigoDistroDetector } from './resources/detectors';
+import packageJson from '../package.json';
```

Comment:
We are actually already reading this data in the detector that sets lumigo.distro.version. Cannot we read the version just once?

Reply:
yes we can!